lib/pages/inbox.dart

```diff
@@ -61,32 +61,39 @@ class InboxPage extends HookWidget {
         ScaffoldMessenger.of(context)
             .showSnackBar(SnackBar(content: Text(e.toString())));
       }
+      await accStore.checkNotifications(accStore.defaultUserData);
     }
 
     return DefaultTabController(
       length: 3,
       child: Scaffold(
         appBar: AppBar(
           title: RadioPicker<String>(
-            onChanged: (val) {
-              selected.value = val;
+            values: accStore.loggedInInstances
+                .expand(
+                  (instanceHost) => accStore
+                      .usernamesFor(instanceHost)
+                      .map((username) => '$username@$instanceHost'),
+                )
+                .toList(),
+            groupValue:
+                '${accStore.defaultUsername}@${accStore.defaultInstanceHost}',
+            onChanged: (value) {
+              final [user, instance] = value.split('@');
```

Comment:
Unrelated, but this particular snippet of code has appeared _a lot_ now. I think we should have an overload of `setDefaultAccount` that takes a full string and does the split internally.